examples/complete/stacks/catalog/terraform/eks_cluster_tmpl.yaml

```diff
@@ -0,0 +1,14 @@
+components:
+  terraform:
+    # Parameterize Atmos component name
+    "eks/cluster-{{ .color }}":
```

Comment:
Can we use `eks-<flavor>/<component>` convention so it gives the intuitive appearance of datadog-agent tied to `eks-blue` e.g. `eks-blue/datadog-agent`. I know we use the other convention in infra-live but we should really migrate away from that

Reply:
@aknysh i think my comment wasnt fully explained. I wasnt remarking on flavor vs color but on where in the component name the flavor/color is located. Please put the color/flavor to the left of the folder directory forward slash so its intuitive that the component belongs to the eks-flavor

Reply:
In comparison

```
core-ue1-prod-eks-blue-cluster
core-ue1-prod-eks-blue-karpenter
core-ue1-prod-eks-blue-fargate-profile
core-ue1-prod-eks-green-cluster
core-ue1-prod-eks-green-karpenter
core-ue1-prod-eks-green-fargate-profile
```

The way ive proposed, you know fargate-profile is part of eks-green whereas the older way we've done it, you dont know if eks/fargate-green is a special flavor of eks/fargate that exists on the eks non-flavor cluster or the fargate non-flavor on the eks flavor

We did this in infralive and other clients but it makes it less intuitive. By codifying it here in the examples and the docs, we give more credibility to this and help encourage its proliferation

Reply:
We spoke about it in the arb on jan 20th which lead to Erik's comment showcasing the catalog import with a parameter that fed the eks attribute to the left of the forward slash.

https://github.com/cloudposse/atmos/issues/307#issuecomment-1398826175

Reply:
what if there was a way to list all the components within a directory so you coupd see all the components in associated with `eks-blue`.

```
atmos describe component eks-blue/*
```

vs

```
atmos describe component eks/*-blue
```

Reply:
What if you had two different actions runners controllers in eks-blue?

The names could be this

```
atmos terraform plan eks-blue/actions-runner-controller-small
atmos terraform plan eks-blue/actions-runner-controller-medium
```

Or this

```
atmos terraform plan eks/actions-runner-controller-blue-small
atmos terraform plan eks/actions-runner-controller-blue-medium
```

What if you used medium instead of blue to denote using medium sized instance sizes for the eks, then you could have this

```
atmos terraform plan eks-medium/actions-runner-controller-small
atmos terraform plan eks-medium/actions-runner-controller-medium
```

Or this

```
atmos terraform plan eks/actions-runner-controller-small-medium
atmos terraform plan eks/actions-runner-controller-medium-medium
```

Reply:
Thank you!